testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -751,6 +720,21 @@ class _GromacsReader_offsets(TestCase):
     ref_volume = 362270.0
     ref_offsets = None
     _reader = None
+    prec = 3
+    
+    @pytest.fixture()
+    def traj(self, tmpdir):
+        shutil.copy(self.filename, str(tmpdir))
+        return os.path.join(str(tmpdir), os.path.basename(self.filename))
+    
+    @pytest.fixture()
+    def trajectory(self, traj):
+        return self._reader(traj)
+
+    @pytest.fixture()
+    def ts(self, trajectory):
+        return trajectory.ts
+    
 
     def setUp(self):
         # since offsets are automatically generated in the same directory
```

Comment:
Put this 2-line comment before the new fixtures – it is important in order to understand why we even bother to copy trajectories around.